### PR TITLE
docs(howto): use outlook with webhosting classic

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: webhosting/classic/how-to/configure-outlook.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  validation: 2021-05-26
+  posted: 2021-05-26
+---
+
+You can read and send your emails using the Microsoft Outlook client. 
+<Message type="requirement">
+
+  - You have an account and are logged into the [Dedibox Console](https://console.online.net)
+  - You have created a webhosting **Classic**
+  - You have [created an email account](/webhosting/classic/how-to/create-email-account)
```
Comment (ldecarvalho-doc) [inline webhosting/classic/how-to/configure-outlook.mdx:14]:
```suggestion
  - You have [created an email account](/webhosting/classic/how-to/create-email-account/)
```

File: webhosting/classic/how-to/configure-outlook.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+dates:
+  validation: 2021-05-26
+  posted: 2021-05-26
+---
+
+You can read and send your emails using the Microsoft Outlook client. 
+<Message type="requirement">
+
+  - You have an account and are logged into the [Dedibox Console](https://console.online.net)
+  - You have created a webhosting **Classic**
```
Comment (ldecarvalho-doc) [inline webhosting/classic/how-to/configure-outlook.mdx:13]: maybe add a link?
```suggestion
  - You have created a **Classic** webhosting 
```

Comment (bene2k1) [reply webhosting/classic/how-to/configure-outlook.mdx:13]: - No link possible here - the offer is EoL and no longer available

File: webhosting/classic/how-to/configure-outlook.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+  - You have an account and are logged into the [Dedibox Console](https://console.online.net)
+  - You have created a **Classic** webhosting 
+  - You have [created an email account](/webhosting/classic/how-to/create-email-account/)
+
+</Message>
+
+<Message type="tip">
+
+We recommend to use the IMAP protocol if you access your mailbox from different devices (PC, Mobile phone, Tablet, …).
```
Comment (RoRoJ) [inline webhosting/classic/how-to/configure-outlook.mdx:20]:
```suggestion
We recommend that you use the IMAP protocol if you want to access your mailbox from different devices (PC, Mobile phone, Tablet, …).
```

File: webhosting/classic/how-to/configure-outlook.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: How to configure Outlook to receive and send email
+description: This page shows you how to Outlook to receive and send email
```
Comment (RoRoJ) [inline webhosting/classic/how-to/configure-outlook.mdx:3]:
```suggestion
description: This page shows you how to use Outlook to receive and send email
```

File: webhosting/classic/how-to/configure-outlook.mdx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+</Message>
+
+You can [configure](https://support.microsoft.com/en-us/office/add-an-email-account-to-outlook-6e27792a-9267-4aa4-8bb6-c84ef146101b) Microsoft Outlook to [check your emails](/webhosting/classic/how-to/check-emails/).
+
+1. Set the following parameters in the configuration wizard, to access your Webhosting Classic mailbox:
+    * IMAP server (incoming mail):
+      * Hostname: imap.online.net
+      * TCP port: 993
+
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply webhosting/classic/how-to/configure-outlook.mdx:30]: nope should be like that

**Pull request merged**